services/tenant-ui/frontend/src/types/acapyApi/acapyInterface.ts

```diff
@@ -3378,7 +3378,7 @@ export interface SchemaStorageRecord {
   /** Schema identifier */
   ledger_id?: string;
   /** (Indy) schema */
-  schema?: object;
+  schema: Schema;
```

Comment:
This might possibly get overwritten on a subsequent generation of this file...
https://github.com/bcgov/traction/tree/main/services/tenant-ui/frontend/src/types#aca-py-api-types

Unless it's due to a change in the actual endpoint swagger definition. Maybe it **should** be updated if it's in a plugin to be represented this way?

Reply:
Oh. I should have read the readme. I was wondering where this file came from.
Hmm. I'll need to think about it. There's probably some work around but I think the definition should definitely have the correct interface here instead of a generic object.

Reply:
Could override/extend it in the other Types file maybe (or locally where it's used) until we check if the Aca-Py Swagger type can be improved.

Reply:
I changed this so there is another type in the types index file that extends the type with the correct interface. May need to do this for other types at some point. It's a little confusing but much safer.